src/Bridge/Symfony/Messenger/DataPersister.php

```diff
@@ -98,4 +86,37 @@ public function remove($data, array $context = [])
                 ->with(new RemoveStamp())
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loop($data, array $context = []): bool
+    {
+        try {
+            $value = $this->getMessengerAttributeValue($this->resourceMetadataFactory->create($context['resource_class'] ?? $this->getObjectClass($data)), $context);
+        } catch (ResourceClassNotFoundException $exception) {
+            return false;
+        }
+
+        return 'persist' === $value || (\is_array($value) && \in_array('persist', $value, true));
+    }
+
+    private function getMessengerAttributeValue(ResourceMetadata $resourceMetadata, array $context = [])
```

Comment:
Missing typehint?

Reply:
No cause the typehint could be boolean, string or array

Reply:
Missing PHPDoc then :stuck_out_tongue: